build.sbt

```diff
@@ -319,7 +321,8 @@ lazy val scioRepl: Project = Project(
 ).settings(
   assemblyJarName in assembly := s"scio-repl-${version.value}.jar"
 ).dependsOn(
-  scioCore
+  scioCore,
+  scioHdfs
```

Comment:
I kinda on the edge on this one - let's merge but - I think we will remove it next time we have an issue with repl and all the HDFS deps.